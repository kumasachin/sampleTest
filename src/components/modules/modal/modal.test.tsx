import React from 'react';
import Modal from './modal';
import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })

it('renders correctly enzyme', () => {
    const wrapper = shallow(<Modal 
        closePopup={()=>{}}
        customerBalanceData={
            {
                fundsHeld: 12,
                currency: "",
                overDraft: 12,
                pendingBalance: 132,
                accountDetails: {
                  sortCode: "123123",
                  accountNumber: "123123123123",
                  id:"123"
                }
            }
        }
        id={1}
    />)

    expect(toJson(wrapper)).toMatchSnapshot();
});