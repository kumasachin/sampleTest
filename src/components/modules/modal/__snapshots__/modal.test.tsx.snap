// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<WithStyles(ForwardRef(Dialog))
  open={true}
>
  <WithStyles(ForwardRef(DialogTitle))>
    Account Number: 123123123123
  </WithStyles(ForwardRef(DialogTitle))>
  <WithStyles(ForwardRef(DialogContent))>
    <ul>
      <li>
        Funds Held:  12
      </li>
      <li>
        Overdraft:  12
      </li>
      <li>
        Pending Balance:  132
      </li>
    </ul>
  </WithStyles(ForwardRef(DialogContent))>
  <WithStyles(ForwardRef(Button))
    onClick={[Function]}
  >
    Close Popup
  </WithStyles(ForwardRef(Button))>
</WithStyles(ForwardRef(Dialog))>
`;
